src/eventgenerator/config/config.go

```diff
@@ -8,41 +8,136 @@ import (
 )
 
 const (
-	DefaultLoggingLevel               = "info"
-	DefaultPollInterval time.Duration = 30 * time.Second
+	DefaultServerPort                int           = 8080
+	DefaultLoggingLevel                            = "info"
+	DefaultPolicyPollerInterval      time.Duration = 40 * time.Second
+	DefaultAggregatorExecuteInterval time.Duration = 40 * time.Second
+	DefaultMetricPollerCount         int           = 20
+	DefaultAppMonitorChannelSize     int           = 200
+	DefaultEvaluationExecuteInterval time.Duration = 40 * time.Second
+	DefaultEvaluatorCount            int           = 20
+	DefaultTriggerArrayChannelSize   int           = 200
 )
 
+type ServerConfig struct {
+	Port int `yaml:"port"`
+}
+
+type LoggingConfig struct {
+	Level string `yaml:"level"`
+}
+
+type DBConfig struct {
+	PolicyDBUrl    string `yaml:"policy_db_url"`
+	AppMetricDBUrl string `yaml:"app_metrics_db_url"`
+}
+
+type AggregatorConfig struct {
+	AggregatorExecuateIntervalInSeconds time.Duration `yaml:"aggregator_execute_interval_in_seconds"`
```

Comment:
This should now be AggregatorExecuateInterval
They can write 10s or whatever units.